streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1495,8 +1502,8 @@ public void shouldOnlyCommitRevokedStandbyTaskAndPropagatePrepareCommitException
         };
         task01.setCommitNeeded();
 
-        taskManager.tasks().put(taskId00, task00);
```

Comment:
We get a read-only view from `tasks()` now, what protects us from accidentally messing with the task container.